apps/src/panels/PanelsLabView.tsx

```diff
@@ -27,6 +31,9 @@ const PanelsLabView: React.FunctionComponent = () => {
   const currentAppName = useAppSelector(
     state => state.lab.levelProperties?.appName
   );
+  const skipUrl = useAppSelector(state => state.lab.levelProperties?.skipUrl);
```

Comment:
Should we provide authoring support for this in levelbuilder? Or maybe we want to hold off since it's a very specific use case?

Reply:
I was thinking about it, and certainly open to it, but happy to do as a follow-up if/when we find a need for it.